core/src/ast_builder/expr/function.rs

```diff
@@ -1,10 +1,7 @@
-use {
-    super::ExprNode,
-    crate::{
-        ast::{DateTimeField, Function},
-        ast_builder::{DataTypeNode, ExprList},
-        result::{Error, Result},
-    },
+use crate::{
+    ast::{DateTimeField, Function},
+    ast_builder::{DataTypeNode, ExprList, ExprNode},
+    result::{Error, Result},
```

Comment:
could we deal these kind of changes in the separate PR?
if we can have these changes in another PR, then we can directly ship these to the main.
the changes themselves are very nice but these are not related to the composite primary key PR?

Reply:
Ok - I will try to do a PR on which we can then re-align this PR. Hopefully, it will not be too much of a mess.

Reply:
Done, see PR #1527